_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Cockpit 248
+author: garrett
+date: '2021-07-07'
+tags: cockpit
+slug: cockpit-248
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 248:
+
+
+## Metrics: Install packages on demand
+
+[Cockpit 247](https://cockpit-project.org/blog/cockpit-247.html) introduced setting up a machine for Grafana metrics. However, this only worked if the `pcp` and `redis` packages were already installed. With this Cockpit version, the options will always work, and missing packages get installed on demand.
+
+![screenshot of install packages on demand]({{ site.baseurl }}/images/248-metrics-install-packages-on-demand.png)
+
+![screenshot of install packages on demand]({{ site.baseurl }}/images/248-metrics-install-packages-on-demand-2.png)
+
+## PAM: Deprecate `pam_cockpit_cert` module
+
+The `pam_cockpit_cert` module is no longer required for client certificate authentication, and has been deprecated.  This release continues to ship a stub module which returns `PAM_IGNORE`, but this stub will be dropped in a future release.  If you modified  `/etc/pam.d/cockpit`, the package upgrade will not automatically update the file -- please remove the relevant entry manually in this case.
+
+## Storage: the Format dialog has been polished
+
+Fields have been re-ordered and warnings and errors are now consistent with Cockpit's common styling.
+
+![screenshot of the format dialog has been polished]({{ site.baseurl }}/images/248-storage-the-format-dialog-has-been-polished.png)
```

Comment:
Not notable enough. Delete.

It's cool and important, but it's just shuffling around form elements in a dialog from a user's point of view. (The rest of the storage fixes that are pending are feature improvements and are notable, however.)